### PR TITLE
windows: Make Qt5 part of the tomviz component

### PR DESCRIPTION
**Pull request by mathturtle**
For some reason this is needed for the NSIS generator to install the libs.
The ZIP generator worked fine without it, which is what I was testing.
There is still an issue with Qt though, starting the application installed
from the NSIS installer says Qt cannot find the 'windows' plugin.

### CODE REVIEW

**Comment by mathturtle** [comment]
@cryos

**Comment by cryos** [comment]
The version I looked at on Mac was the ZIP file for the 64 bit Windows build, and I still only see QtTesting.dll (none of the other Qt DLLs appear to be present). It isn't clear why this change should be needed. I would prefer to work on something that will produce a valid package, but will bear this in mind - the platform plugin is required for Qt 5 on all platforms.

**Comment by mathturtle** [comment]
@cryos With the additional commit that takes care of the windows platform plugin, this makes the windows binaries work again.

File: projects/win32/qt.use.system.cmake [hunk 1]
```diff
@@ -39,4 +38,9 @@ foreach(dir IN LISTS qt5_libdirs)
     endif()
   endforeach()
 endforeach()
-")
+" COMPONENT "tomviz")
+
+get_target_property(qtWindowsPluginLocation Qt5::QWindowsIntegrationPlugin LOCATION)
+install(FILES ${qtWindowsPluginLocation}
+        DESTINATION "bin/platforms"
+	COMPONENT "tomviz")
```
Comment (cryos) [inline projects/win32/qt.use.system.cmake]: Can you update this to make the indentation consistent please.

**Comment by cryos** [comment]
Now it looks like it would be worth running an updated dashboard, let's fix the whitespace issue and get it merged. The broader Qt 5 ParaView patch is still pending, I will check on that.

**Comment by mathturtle** [comment]
Apparently gVim was doing something funny with tabs instead of spaces...  Whitespace fixed.

**Pull request merged**